ui/apps/search/src/components/routes/entry-renderer.tsx

```diff
@@ -112,8 +112,8 @@ const EntryCardRenderer = (props: EntryCardRendererProps) => {
               locale={locale}
               moderatedContentLabel={t('This content has been moderated')}
               profileAnchorLink={'/@akashaorg/app-profile'}
-              repliesAnchorLink={`/@akashaorg/app-akasha-integration/${
-                itemType === EntityTypes.REFLECT ? 'reflect' : 'beam'
+              reflectAnchorLink={`/@akashaorg/app-akasha-integration/${
+                itemType === EntityTypes.REFLECT ? 'reflection' : 'beam'
```

Comment:
I think also here it can replaced with an imported route variable

Reply:
I have thought about this as well and like Sever mentioned we couple one app inside another. Do you have in mind another approach that doesn't do this coupling ?